packages/internal/src/__tests__/build_api.test.ts

```diff
@@ -148,3 +336,97 @@ test('jest mock statements also handle', () => {
   // Step 3: check that output has correct jest.mock path
   expect(outputForJest).toContain('jest.mock("../../lib/dog"')
 })
+
+test('core-js polyfill list', () => {
+  const { list } = compat({
+    targets: { node: TARGETS_NODE },
+    version: 3,
+  })
```

Comment:
These are the same settings passed to plugin-transform-runtime:

https://github.com/redwoodjs/redwood/blob/0e087ade5ad689efdeedd5e846749a17158c172b/packages/internal/src/build/babel/api.ts#L91-L102

Reply:
What do you say we put these settings in a variable and just import it in this test and also use it to configure transform runtime? That way we always know they're using the same stuff.

Reply:
Yeah that's a good call